packages/@sanity/default-layout/src/DefaultLayout.css

```diff
@@ -62,13 +62,16 @@
 }
 
 .loadingScreen {
+  /* Override Sanity UI */
```

Comment:
Maybe add a note about *why* we're overriding Sanity UI and how long we need to keep it or when we can remove it? 😊